lib-ee/emqx_ee_bridge/test/emqx_ee_bridge_pgsql_SUITE.erl

```diff
@@ -45,14 +45,20 @@ groups() ->
     [
         {tcp, [
             {group, with_batch},
-            {group, without_batch}
+            {group, without_batch},
+            {group, matrix},
+            {group, timescale}
```

Comment:
I think that those sub-types should be the outer groups of tests, e.g.:

```
└── matrix
    └── tcp
        └── with_batch
```

So they would already have a TLS/TCP/Batching testing matrix set up.